src/v2/Apps/Partner/Components/PartnerArticles/helpers/articlesCard.ts

```diff
@@ -0,0 +1,28 @@
+import _ from "underscore"
+import _s from "underscore.string"
```

Comment:
I'm not sure what the deal is with underscore/underscore.string anymore? Do we use those in new code? Lodash only? It looks like both of these are pretty simple functions that don't require the use of a lib anyway?

Reply:
Well, I don't know actually what libs is ok to use on Artsy and what are not. I spoke with Sarah regarding the logic, she said just do the same as it was previously, so I did

Reply:
I think in general: use lodash if you need to, and avoid underscore which is a relic from the older codebase.

As far as this code goes:
```tsx
authorNameObj.authorName = _s.toSentence(
  _.pluck(contributingAuthors, "name")
)

// Is equivalent
contributingAuthors.map(({ name }) => name).join(', ').replace(/,\s([^,]+)$/, ', and $1')
```

I'd also add some types to this function.

Reply:
Sorry if I confused anything @ivan-badyulya ! I agree with @dzucconi -- we're trying to remove dependencies on underscore when we rebuild pages, so I recommend going with his suggestion.

Reply:
Hey @sweir27 @dzucconi ! I believe I fixed it. Could you look into?